server/base/src/main/java/org/apache/accumulo/server/master/LiveTServerSet.java

```diff
@@ -120,11 +120,17 @@ public TabletServerStatus getTableMap(boolean usePooledConnection) throws TExcep
       if (usePooledConnection == true)
         throw new UnsupportedOperationException();
 
+      long start = System.currentTimeMillis();
```

Comment:
Using `System.nanoTime()` would be better here. More accurate and not subject to system clock change.